src/components/atoms/Dates/styles.module.scss

```
@@ -0,0 +1,10 @@
+@use '../../../styles/mixin.scss';
+@use '../../../styles/variables.module.scss' as variables;
+
+.date_text {
+  font-size: 0.875rem;
+  line-height: 1.5;
```

⚙️ [nits]
WideCard の日付についてですが、太字の指定なので設定してください！

```suggestion
.date_text {
  font-size: 0.875rem;
  line-height: 1.5;
  font-weight: 700;
```